server/Makefile

```diff
@@ -691,6 +691,19 @@ config-saml: ## Configures SAML.
 	@sed -i'' -e 's|"AssertionConsumerServiceURL": ".*"|"AssertionConsumerServiceURL": "http://localhost:8065/login/sso/saml"|g' ../config/config.json
 	@sed -i'' -e 's|"IdpCertificateFile": ".*"|"IdpCertificateFile": "saml-idp.crt"|g' ../config/config.json
 
+config-openid: ## Configures OpenID.
+	@echo Setting up configuration for local OpenID with keycloak, please ensure your keycloak is running on http://localhost:8484
+
+	# Check if jq is installed
+	@jq --version > /dev/null 2>&1 || (echo "jq is not installed. Please install jq to continue." && exit 1)
+
+	TMPDIR=$(mktemp -d)
+	jq --slurp '.[0] * .[1]' config/config.json build/docker/keycloak/openid.mmsettings.json > ${TMPDIR}/config.json
+	cp ${TMPDIR}/config.json ./config/config.json
+	rm ${TMPDIR}/config.json
```

Comment:
Nice use of `jq` :+1: Do you also want to migrate `config-saml` and `config-ldap` as part of this PR?

Reply:
I was about to do it but I saw that `config-saml` and `config-ldap` use `../config/config.json` for the config file path, while my environment uses `config/config.json`. Which one is correct? I want to unify usage. cc: @BenCookie95

Reply:
https://github.com/mattermost/mattermost/pull/26518/commits/d78d662890677c26f66e84fafd82b9bd01a5a957